python/openvino_tokenizers/utils.py

```diff
@@ -290,3 +296,64 @@ def quote_meta(unquoted: Union[str, bytes]) -> str:
             symbols.append("\\")
         symbols.append(char)
     return "".join(symbols)
+
+
+def to_bytes(number: int) -> bytes:
+    return number.to_bytes(4, "little")
+
+
+class UnpackedOutputs:
+    _outputs = None
+    def __init__(self, outputs):
+        self._outputs = outputs
+
+    def outputs(self) -> List:
+        if self._outputs:
+            return self._outputs
+        else:
+            return []
+
+
+def create_unpacked_string(strings: Iterable[str]) -> UnpackedOutputs:
+    """
+    Convert any list of strings to U8/1D numpy array with begins, ends, and chars
+    """
+    strings = list(strings)
+    batch_size = len(strings)
+    if batch_size == 0:
+        return np.frombuffer(to_bytes(0), np.uint8)
+
+    buffer = BytesIO()
+    buffer.write(to_bytes(batch_size))
+    begins = BytesIO()
+    ends = BytesIO()
+    chars = BytesIO()
+    offset = 0
+
+    for string in strings:
+        byte_string = string.encode("utf-8") if isinstance(string, str) else string
+        length = len(byte_string)
+
+        begins.write(to_bytes(offset))
+        offset += length
+        ends.write(to_bytes(offset))
+        chars.write(byte_string)
+
+    begins = np.frombuffer(begins.getvalue(), np.int32)
+    ends = np.frombuffer(ends.getvalue(), np.int32)
+    chars = np.frombuffer(chars.getvalue(), np.uint8)
+    
+    return UnpackedOutputs([Constant(Tensor(x)).output(0) for x in [begins, ends, chars]])
+
+
+def create_str_constant(strings: Iterable[Union[str, bytes]]) -> Constant:
+    """
+    Create a string constant from strings/bytes.
+    """
+    strings = list(strings)
+    batch_size = len(strings)
+    if batch_size == 0:
+        return Constant(ov.Type.string, [])
+
+    strings = [bytes(string, "utf-8") if isinstance(string, str) else string for string in strings]
+    return Constant(Tensor(np.array(strings)))
```

Comment:
Duplicates `create_string_constant_node` method. One of them should be deleted.

Reply:
removed duplication